Sources/UIWindow.swift

```diff
@@ -12,6 +12,7 @@ public class UIWindow: UIView {
     }
 
     open func makeKeyAndVisible() {
+        self.safeAreaInsets = UIWindow.getSafeAreaInsets()
```

Comment:
we consume this from here
https://github.com/flowkey/NativePlayer/blob/master/FlowkeyPlayer/UIView%2BsafeArea.swift#L14